tests/form.js

```diff
@@ -361,7 +361,7 @@ define([
                 [{id: "old", src: "new://3"}, "old-1", "src"],
                 [{id: "old", src: "old://"}, "old", "src"],
                 [{id: "any", src: "old://"}, "old", "src"],
-                [{id: "known", src: null}, "known", "known://"],
+                [{id: "known", src: undefined}, "known", "known://"],
```

Comment:
@millerdev I couldn't find a place where this would actually be `undefined`

The reason I changed it was for (http://underscorejs.org/#1.5.0):

> The defaults function now only overrides undefined values, not null ones.